AirCasting/AuthViews/UserAuthenticationSession.swift

```diff
@@ -29,8 +29,11 @@ final class UserAuthenticationSession: ObservableObject {
     }
 
     func deauthorize() throws {
-        self.token = nil
+        URLSession.shared.getAllTasks { tasks in
+            tasks.forEach { $0.cancel() }
+        }
```

Comment:
You've already done that in the logout function, is there a reason to repeat canceling all tasks here?

Reply:
If there would be a pending call to api that synchronises the sessions there is no check if the user is logged in or not. so if we wouldn't cancel it the returned sessions would again be added into the database.